src/main/java/by/home/port/Port.java

```diff
@@ -0,0 +1,50 @@
+package by.home.port;
+
+import java.util.List;
+import java.util.concurrent.Semaphore;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
+public class Port {
+    private final static int DOCKS_NUMBER = 3;
+    List<Dock> docks;
+    private Lock lock = new ReentrantLock();
+    private Semaphore semaphore = new Semaphore(DOCKS_NUMBER);
+    private static volatile Port port;
```

Comment:
instance

Reply:
i thought. this name only for books for instances. but ok